headers/types/common/common.h

```diff
@@ -128,6 +128,19 @@ struct overlay_load_entry {
 };
 ASSERT_SIZE(struct overlay_load_entry, 16);
 
+// Struct containing information about an overlay. The entries are copied from y9.bin.
+struct overlay_info_entry {
+    int overlay_id;                // 0x0
+    int ram_address;               // 0x4
+    int size;                      // 0x8
+    int bss_size;                  // 0xC: "Size of BSS data region"
```

Comment:
I copied that literally from the No$GBA docs

Reply:
Is there a reason to emphasize that though? It's a fairly generic description. Unless you're concerned about plagiarism, why not just

```suggestion
    int bss_size;                  // 0xC: Size of BSS data region
```

Reply:
Just to show that I copied it literally and have no idea what it means myself lol
I could remove them if you like it better that way, I don't really mind.

Reply:
Yeah, that sounds good. The [BSS segment](https://en.wikipedia.org/wiki/.bss) is a generic computing concept. It's a lot like the [`.data` segment](https://en.wikipedia.org/wiki/Data_segment), except the former is for globals _without_ a declared value, while the latter is for globals _with_ a declared value:

```c
int bss;
int data = 1;
```

With declared values, you have to store those values in the binary somewhere (the `.data` section). With undeclared values, you can omit them from the binary itself to save space, since you can "reconstruct" it in memory at runtime when you first load the executable as a section of all zeroes. So the binary just has to store the BSS size.